app/models/goal.rb

```diff
@@ -1,14 +1,14 @@
 class Goal < ApplicationRecord
   include Searchable
   
-  AVAILABLE_COLORS = %w[[Purple rgb(140,8,246)],
-                      [Blue rgb(17,85,253)],
-                      [Red rgb(255,18,12)],
-                      [Pink rgb(255,125,253)],
-                      [Cyan rgb(1,240,253)],
-                      [Yellow rgb(255,255,33)],
-                      [Gold rgb(252,206,24)],
-                      [Midnight blue rgb(14,31,82)]]
+  AVAILABLE_COLORS = [['Purple', 'rgb(140,8,246)'],
```

Comment:
Not sure where this constant is being used, but you could probably move it to the helper which is a more appropriate place for UI related data.